lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -521,14 +522,20 @@ def build_image(self):
         for line in self.client.build(**params):
             # line = json.loads(line)
             self.log(line, pretty_print=True)
+            if "stream" in line:
+                build_output.append(line["stream"])
             if line.get('error'):
                 if line.get('errorDetail'):
                     errorDetail = line.get('errorDetail')
-                    self.fail("Error building %s - code: %s message: %s" % (self.name,
-                                                                            errorDetail.get('code'),
-                                                                            errorDetail.get('message')))
+                    self.fail(
+                            "Error building %s - logs: %s, code: %s message: %s" % (
```

Comment:
I'd prefer to have logs last in the error message, since they're going to be by far the most verbose.